doc/guides/dts/index.rst

```diff
@@ -196,61 +269,58 @@ setting the bus speed:
+Devicetree is currently supported on all embedded targets except posix
 (boards/posix) and qemu_x86_64 (boards/x86_64).
 
 Adding support for a board
 **************************
 
-Adding device tree support for a given board requires adding a number of files.
+Adding devicetree support for a given board requires adding a number of files.
 These files will contain the DTS information that describes a platform, the
-YAML descriptions that define the contents of a given Device Tree peripheral
-node, and also any fixup files required to support the platform.
+bindings in YAML format, and any fixup files required to support the platform.
 
-When writing Device Tree Source files, it is good to separate out common
-peripheral information that could be used across multiple SoC families or
-boards.  DTS files are identified by their file suffix.  A .dtsi suffix denotes
-a DTS file that is used as an include in another DTS file.  A .dts suffix
-denotes the primary DTS file for a given board.
+It is best practice to separate common peripheral information that could be
+used across multiple cores, SoC families, or boards in :file:`.dtsi` files,
+reserving the :file:`.dts` suffix for the primary DTS file for a given board.
 
-The primary DTS file will contain at a minimum a version line, optional
-includes, and the root node definition.  The root node will contain a model and
-compatible that denotes the unique board described by the .dts file.
+Devicetree Source File Template
+===============================
 
-Device Tree Source File Template
-================================
+A board's :file:`.dts` file contains at least a version line, optional
+includes, and a root node definition with ``model`` and ``compatible``
+properties. These property values denote the particular board.
 
 .. code-block:: none
 
-  /dts-v1/;
+   /dts-v1/;
 
-  / {
-          model = "Model name for your board";
-          compatible = "compatible for your board";
-          /* rest of file */
-  };
+   #include <vendor/soc.dtsi>
 
+   / {
+           model = "Human readable board name";
+           compatible = "vendor,soc-on-your-board's-mcu";
+           /* rest of file */
+   };
 
-One suggestion for starting from scratch on a platform/board is to examine other
-boards and their device tree source files.
+You can use other board :file:`.dts` files as a starting point.
 
 The following is a more precise list of required files:
 
 * Base architecture support
 
   * Add architecture-specific DTS directory, if not already present.
     Example: dts/arm for ARM.
-  * Add target specific device tree files for base SoC.  These should be
-    .dtsi files to be included in the board-specific device tree files.
+  * Add target specific devicetree files for base SoC.  These should be
+    .dtsi files to be included in the board-specific devicetree files.
   * Add target specific YAML binding files in the dts/bindings/ directory.
     Create the yaml directory if not present.
```

Comment:
Nit: Could highlight stuff like `dts/arm` and `.dtsi` above maybe.

Reply:
Sorry, I don't get it -- what do you mean by "highlight"? Emphasize (in italics, bold, etc.), wrap in `:file:`, add a `:ref:` to somewhere earlier in the doc?